py/torch_tensorrt/dynamo/conversion/impl/quantize.py

```diff
@@ -26,20 +27,51 @@ def quantize_fp8(
         input_tensor.dtype == trt.float32 or input_tensor.dtype == trt.float16
     ):
         raise ValueError(
-            f"quantize_fp8 converter received an input of {input_tensor.dtype} type. Supported types: float32 | float16"
+            f"quantize {quantize_type} converter received an input of {input_tensor.dtype} type. Supported types: float32 | float16"
+        )
+    if quantize_type not in ["fp8", "int8"]:
+        raise ValueError(
+            f"{quantize_type=} is not supported. Supported types: fp8 | int8"
         )
-
     scale = get_trt_tensor(ctx, scale, name + "_scale")
     # Add Q node
     quantize_layer = ctx.net.add_quantize(input_tensor, scale)
-    quantize_layer.set_output_type(0, trt.DataType.FP8)
+    if quantize_type == "int8":
+        quantize_layer.set_output_type(0, trt.DataType.INT8)
+    else:
+        quantize_layer.set_output_type(0, trt.DataType.FP8)
     set_layer_name(quantize_layer, target, name + "_quantize", source_ir)
     q_output = quantize_layer.get_output(0)
     # Add DQ node
     dequantize_layer = ctx.net.add_dequantize(q_output, scale)
     set_layer_name(dequantize_layer, target, name + "_dequantize", source_ir)
-    # Set DQ layer precision to FP8
-    dequantize_layer.precision = trt.DataType.FP8
+    if quantize_type == "int8":
+        dequantize_layer.precision = trt.DataType.INT8
+    else:
+        # Set DQ layer precision to FP8
+        dequantize_layer.precision = trt.DataType.FP8
     dq_output = dequantize_layer.get_output(0)
 
     return dq_output
+
+
+def quantize_fp8(
```

Comment:
Dont think this indirection is necessary, aten can just pass the enum for the quantization type since its already specialized there